README.md

```diff
@@ -43,199 +43,30 @@ We have proposed a new WASI issue [wasi-bpf](https://github.com/WebAssembly/WASI
 
 ## 🔨 Examples
 
-See the [examples](examples) directory for examples of eBPF programs written in C, Rust and compiled to WASM.
+See the [examples](examples) directory for examples of eBPF programs written in C, Rust, Go and compiled to WASM.
 
-- [bootstrap](examples/bootstrap) and [runqlat](examples/runqlat) `tracing examples`
-- [rust-bootstrap](examples/rust-bootstrap) `tracing examples`
-- [lsm](examples/lsm) `security example`
+- [bootstrap](examples/bootstrap) and [rust-bootstrap](examples/rust-bootstrap) `tracing examples`
+- [runqlat](examples/runqlat) `tracing examples`
+- [go-execve](examples/go-execve) `tracing examples`
+- [lsm](examples/lsm) and  `security example`
 - [sockfilter](examples/sockfilter) `networking example`
 - [sockops](examples/sockops) `networking example`
```

Comment:
Maybe better with the following format?

- [sockops](examples/sockops): Add the pid int tcp option in syn packet.

Reply:
Yes! Thank you for notification!